R/fileupload.R

```diff
@@ -52,7 +78,8 @@ FileUploadOperation <- R6Class(
       .currentFileInfo <<- file
       .pendingFileInfos <<- tail(.pendingFileInfos, -1)
 
-      filename <- file.path(.dir, as.character(length(.files$name)))
+      basename <- .currentFileInfo$name
+      filename <- file.path(.dir, sanitizeFileName(basename, as.character(length(.files$name))))
```

Comment:
That's weird. Through the Win32 API, the pathname limit is 260 minus 1 for `\0`. https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#maxpath